client/my-sites/site-indicator/site-indicator.jsx

```diff
@@ -124,8 +133,11 @@ export default React.createClass( {
 		}.bind( this ), 15000 );
 	},
 
-	handlePluginsUpdate() {
+	handlePluginsUpdate( event ) {
 		window.scrollTo( 0, 0 );
+		if ( this.props.onSelect ) {
+			this.props.onSelect( event );
```

Comment:
Something I like to do in cases like this is to eliminate the presence-checking by setting a default `noop` on the prop…

```js
import { noop } from 'lodash';

getDefaultProps: () => ( {
	onSelect: noop
} )
```

This way, we can safely call `this.props.onSelect( event )` without the conditional.